shared/util.js

```diff
@@ -832,7 +449,7 @@ const UIStrings = {
   /** Label indicating that Lighthouse throttled the page using custom throttling settings. */
   runtimeCustom: 'Custom throttling',
 };
-Util.UIStrings = UIStrings;
```

Comment:
These strings look specific to the report renderer. Would it make more sense for them to live in `ReportUtils`?

Reply:
Made this change, but it required not initializing `strings` as a copy of UIStrings since that made a cyclic dependency between report-utils.js and report-globals.js. Only matter for test code, which had to be updated.

I then saw a nice cleanup to the globals to provide a `apply` function to set them all up.